### ISSUE TITLE
Sync read state to Discord.

### ISSUE THREAD

**Issue by flowbish** [khyperia/weechat-discord#9]
Sync the read state of the weechat buffer back to Discord. Should be a user-configurable option.

**Comment by khyperia** [comment]
Okay. Wow. I just found [this PR](https://github.com/hammerandchisel/discord-api-docs/pull/115). I'm 900% done with Discord at this point, they clearly do not intend the API to be used to implement clients, and AFAIK, they don't have documentation for the thing clients are _actually_ supposed to use.

**Comment by Aanok** [comment]
They do have a very strong focus on their own client, beyond the usual "we don't guarantee unofficial clients will keep working between revisions of the API". BetterDiscord has had its fair share of troubles too, for instance.

It's very possible they'll completely make the API for bots only at some point, instead of just obfuscating docs. It's not exactly FOSS, as friendly and quirky as they paint themselves (or perhaps in perfect agreement with that), they're ultimately a for profit company.

But I'd rather this project didn't die. Weecord has about one tenth the RSS footprint of the official client on my machine (~50MB vs a ridiculous 450MB. A browser tab sits around 200 but then there's the rest of the browser...). That's a tremendous upside and, at least for me, it makes the choice a no-brainer.

**Comment by khyperia** [comment]
> But I'd rather this project didn't die.

Unfortunately, this is a case of maintainer burnout. But - you're free to fork it!

> at least for me

Exactly :) - it's most _definitely_ not for me.

Ultimately, it comes down to this project being a side hobby I do in my free time, for free. My daytime job has me dealing with a lot of terrible things, and a girl can only deal with a certain amount of terrible things every day 😊

**Comment by Aanok** [comment]
Fair enough, fair enough. Thanks for what you've done so far, then, and best of luck!
